raiden_libs/network/matrix/client.py

```diff
@@ -349,3 +374,12 @@ def set_account_data(self, type_: str, content: Dict[str, Any]) -> dict:
         """ Use this to set a key: value pair in account_data to keep it synced on server """
         self.account_data[type_] = content
         return self.api.set_account_data(quote(self.user_id), quote(type_), content)
+
+
+# Monkey patch matrix User class to provide nicer repr
+@wraps(User.__repr__)
+def user__repr__(self):
```

Comment:
The class is initialized inside Matrix's code? Is that why we can't use a subclass? What about contributing upstream instead of monkey patching locally, specially since this looks just like a cosmetic change?

Reply:
Yes, yes. We plan to do that with everything, once these changes settle a little.